### PR TITLE
tests: Add docker diff test

### LINKED ISSUE #450: add docker diff test
add integration docker diff test

### PR DESCRIPTION
**Pull request by GabyCT**
Add integration docker diff test

Fixes #450

Signed-off-by: Gabriela Cervantes Tellez <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: integration/docker/diff_test.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	AfterEach(func() {
+		Expect(RemoveDockerContainer(id)).To(BeTrue())
+		Expect(ExistDockerContainer(id)).NotTo(BeTrue())
+	})
+
+	Context("inspect changes in a container", func() {
+		It("should retrieve the change", func() {
+			stdout, _, exitCode := DockerDiff(id)
+			Expect(exitCode).To(Equal(0))
+			Expect(stdout).To(ContainSubstring("FirstDirectory"))
```
Comment (devimc) [inline integration/docker/diff_test.go]: ```FirstDirectory``` should be a constant

File: integration/docker/diff_test.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+var _ = Describe("diff", func() {
+	var (
+		id string
+	)
+
+	BeforeEach(func() {
+		id = randomDockerName()
+		_, _, exitCode := DockerRun("--name", id, "-d", Image, "sh")
+		Expect(exitCode).To(Equal(0))
+		_, _, exitCode = DockerExec(id, "sh", "-c", "mkdir FirstDirectory")
```
Comment (devimc) [inline integration/docker/diff_test.go]: ```DockerExec("mkdir", DIR)```

**Comment by GabyCT** [comment]
@devimc changes applied

**Comment by chavafg** [comment]
lgtm

**Pull request merged**